yrs/src/block_store.rs

```diff
@@ -173,12 +173,11 @@ impl Decode for Snapshot {
 /// A resizable list of blocks inserted by a single client.
 pub(crate) struct ClientBlockList {
     list: Vec<Box<block::Block>>,
-    integrated_len: usize,
```

Comment:
As part of fixing an issue it turned out that `intergrated_len` is not necessary.